#! /bin/sh /usr/share/dpatch/dpatch-run
## add_classpath_to_manifest.dpatch by Florian Weimer <fweimer@bfk.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: add Class-Path entry to the manifest, so packages using
## DP: httpunit don't have to worry about httpunit's internal dependencies.

@DPATCH@
diff -urNad httpunit-1.7+dfsg~/build.xml httpunit-1.7+dfsg/build.xml
--- httpunit-1.7+dfsg~/build.xml	2008-05-20 09:55:36.000000000 -0430
+++ httpunit-1.7+dfsg/build.xml	2009-07-25 14:30:26.000000000 -0430
@@ -241,6 +241,9 @@
 Build-Time: ${TSTAMP}
 </echo>
     <jar jarfile="${lib.dir}/${name}.jar" manifest="${build.dir}/info.txt">
+        <manifest>
+            <attribute name="Class-Path" value="jtidy.jar nekohtml.jar xercesImpl.jar"/>
+        </manifest>
         <fileset dir="${build.classes}" includes="com/**"/>
         <fileset dir="META-INF" includes="*.dtd"/>
     </jar>
